src/internal/svg/disconnectSvg.tsx

```diff
@@ -6,16 +6,16 @@ export const disconnectSvg = (
     aria-label="ock-disconnect-svg"
     width="100%"
     height="100%"
-    viewBox="0 0 20 20"
+    viewBox="0 0 16 20"
```

Comment:
curious why you updated this to 16 here ? are the other SVGs the same dimensions ?

Reply:
`viewBox 0 0 16 20 ` was specified in this new Svg - when downloading the image and displaying the code. 

The image is off center when using `0 0 20 20`.